### ISSUE TITLE
Is there a recommended implementation of yuv2RGB for the current torch?

### ISSUE THREAD

**Issue by tensorflowt** [pytorch/pytorch#98077]
### 🚀 The feature, motivation and pitch

I need to convert the YUV420SP image to RGB, and then I built a network by torch. However, when I turned to the onnx frame model at last, I found it was very large. Is there a good way to do this?

![YUV420SP2RGB_BY_TORCH](https://user-images.githubusercontent.com/37217594/229105058-f548d1ec-445c-4437-934b-c6ea53ec277c.png)

cc @albanD @mruberry @jbschlosser @walterddr @mikaylagawarecki

**Comment by pat749** [comment]
You can try Pruning in this technique Pruning removes unnecessary weights and connections from your model, reducing its size without significantly affecting accuracy. PyTorch also has built-in support for pruning, and you can use it to remove unimportant parameters from your model before converting it to ONNX.

**Comment by ezyang** [comment]
torch.compile may do a good job compiling this

**Comment by BowenBao** [comment]
@tensorflowt thanks for reporting. Can confirm onnx export is generating ten thousand nodes for `repeat_interleave`. cc @justinchuby if atenlib can help.

**Comment by BowenBao** [comment]
Curious is there decomposition for `repeat_interleave` @ezyang ? I don't know where to find it, but I suppose there should be one? since the op is not part of core aten IR.

**Comment by ezyang** [comment]
I don't see one. So could be an oversight in the core aten IR list / we need a decomp for it.

**Comment by ilyasher** [comment]
@tensorflowt Can you retry onnx export with latest `main` branch?

**Comment by MaanavD** [comment]
Completed with #100575
